docs/recent_activity/recent_activity.md

```diff
@@ -0,0 +1,54 @@
+---
+description: Monitor recent activity logged actions.
+---
+
+# Recent activity log
+
+## Activity list
+
+On the Back Office, in Admin -> Activity list, you can consult what happens recently on the database.
+
+By default, the following actions are logged:
+
+- [Content](content_items.md) [create](create_edit_content_items.md#create-content-items), [publish](publish_instantly.md#publish-content-instantly), update, trash, recover, [delete](copy_move_hide_content.md#multi-file-move), delete translation, hide and reveal
+- Location create, delete, [hide](manage_locations_urls.md#hide-locations), reveal, update, move, [swap](manage_locations_urls.md#swap-locations) and subtree copy
+- Product [create](create_edit_product.md), update and delete
+- Product variant [create](work_with_product_variants.md#generate-variants), update and [delete](work_with_product_variants.md#delete-variants)
+- [Site](work_with_sites.md) [create](work_with_sites.md#create-a-website), [update](work_with_sites.md#edit-an-existing-website) and [delete](work_with_sites.md#delete-an-existing-website)
+
+By default, log entries are kept 30 days.
+
+Log entries are grouped by logical bonds like events, batches, or sessions.
```

Comment:
I feel like I know what you're trying to say, but for an external persona this feels like it will be hard to understand. An example would probably be helpful.
```suggestion
Log entries are grouped by logical bonds like web requests, events, batches, or sessions.
```